MetaCocktailsSwiftData/Model/SearchTags/Tags.swift

```diff
@@ -6,80 +6,25 @@
-    static func createComponentArray() ->  [CocktailComponent] {
-        var array = [CocktailComponent]()
-        for flavor in Flavor.allCases {
-            array.append(CocktailComponent(for: flavor))
-        }
-//        for base in Booze.allCases {
-//            array.append(CocktailComponent(for: base))
-//        }
-        for profile in Profile.allCases {
-            array.append(CocktailComponent(for: profile))
-        }
-        for texture in Texture.allCases {
-            array.append(CocktailComponent(for: texture))
-        }
-        for style in Style.allCases {
-            array.append(CocktailComponent(for: style))
-        }
-        for liqueur in Spirit.Liqueur.allCases {
-            array.append(CocktailComponent(for: liqueur, category: SpiritCategoryName.liqueur))
-        }
-        for agave in Spirit.Agave.allCases {
-            array.append(CocktailComponent(for: agave, category: SpiritCategoryName.agave))
-        }
-        for amari in Spirit.Amari.allCases {
-            array.append(CocktailComponent(for: amari, category: SpiritCategoryName.amari))
-        }
-        for bitters in Spirit.Bitters.allCases {
-            array.append(CocktailComponent(for: bitters, category: SpiritCategoryName.bitters))
-        }
-        for brandy in Spirit.Brandy.allCases {
-            array.append(CocktailComponent(for: brandy, category: SpiritCategoryName.brandy))
-        }
-        for fortifiedWine in Spirit.FortifiedWine.allCases {
-            array.append(CocktailComponent(for: fortifiedWine, category: SpiritCategoryName.fortifiedWine))
-        }
-        for gin in Spirit.Gin.allCases {
-            array.append(CocktailComponent(for: gin, category: SpiritCategoryName.gin))
-        }
-        for vodka in Spirit.Vodka.allCases {
-            array.append(CocktailComponent(for: vodka, category: SpiritCategoryName.vodka))
-        }
-        for rum in Spirit.Rum.allCases {
-            array.append(CocktailComponent(for: rum, category: SpiritCategoryName.rum))
-        }
-        for other in Spirit.Other.allCases {
-            array.append(CocktailComponent(for: other, category: SpiritCategoryName.other))
-        }
-        for whiskies in Spirit.Whiskies.allCases {
-            array.append(CocktailComponent(for: whiskies, category: SpiritCategoryName.whiskies))
-        }
-        for wine in Spirit.Wine.allCases {
-            array.append(CocktailComponent(for: wine, category: SpiritCategoryName.wine))
-        }
-        return array
-    }
+    var agave: [BoozeEnums.Agave]?
+    var amari: [BoozeEnums.Amari]?
+    var bitters: [BoozeEnums.Bitters]?
+    var brandy: [BoozeEnums.Brandy]?
+    var fortifiedWine: [BoozeEnums.FortifiedWine]?
+    var gin: [BoozeEnums.Gin]?
+    var liqueur: [BoozeEnums.Liqueur]?
+    var other: [BoozeEnums.Other]?
+    var rum: [BoozeEnums.Rum]?
+    var vodka: [BoozeEnums.Vodka]?
+    var whiskies: [BoozeEnums.Whiskies]?
+    var wine: [BoozeEnums.Wine]?
     
```

Comment:
Since we're trying to get away from this type of thing - what's the reason we can't just have a single variable here?

var booze: [Booze]?